services/topics/libs/topics-lib.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable no-console */
+const _ = require("lodash");
+import { Kafka, ResourceTypes } from "kafkajs";
+
+export async function createTopics(brokerString, topicNamespace, topicsConfig) {
```

Comment:
Docstring here would be helpful—from context, this function creates Kafka topics? Is this a bootstrap tool?

Reply:
Docstring added

Reply:
ty!